### ISSUE TITLE
Windfury extra attacks consume flurry charges

### ISSUE THREAD

**Issue by Ridley11** [elysium-project/server#2617]
Hello

The shaman enhancement talent flurry's charges are being consumed by the windfury extra attacks, this is the incorrect behaviour according to http://vanilla-wow.wikia.com/wiki/Windfury_Weapon and my own memory.

For a bit more information I am using rank 1 of windfury, and have 4/5 points in flurry. Can confirm this behaviour occurs on 1-3 as well.

Thanks.

Update: Now have 5/5 of flurry and it is behaving as normal. Windfury is no longer consuming the flurry charges.

So basically this means that 1-4 of flurry causes this problem, as I'm guessing the windfury attacks only check for 5/5 version of the flurry buff, which thinking about it makes sense, since it's rare that a shaman would have windfury before max flurry as they max it at 29. 
In my case I put 5 points into elemental first which meant that I had windfury just as I started getting flurry. Also explains why no one else had mentioned this.

**Comment by lovemonkeyz** [comment]
You are correct. Flurry charges should only be consumed on auto attacks or "on next attack" abilities (of which shamans have none). Windfury crits can proc flurry, windfury should not consume flurry.

**Comment by Ridley11** [comment]
Update: Now have 5/5 of flurry and it is behaving as normal. Windfury is no longer consuming the flurry charges.
So basically this means that 1-4 of flurry causes this problem, as I'm guessing the windfury attacks only check for 5/5 version of the flurry buff, which thinking about it makes sense, since it's rare that a shaman would have windfury before max flurry as they max it at 29. In my case I put 5 points into elemental first which meant that I had windfury just as I started getting flurry. Also explains why no one else had mentioned this.
